test/test_exception.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+import pytest
+
+from welkin.exceptions import WelkinHTTPError
+
+
+@pytest.mark.vcr()
+def test_exception(client):
```

Comment:
would we want to have a test actually against Welkin or does it not matter?

Reply:
`test_exception` does actually hit the Welkin API at `https://api.live.welkincloud.io/{{ tenant }}/admin/users/me`, which is a real endpoint that IIRC isn't allowed for API clients. I'm pointing `test_exception_no_json` because I don't know of a Welkin endpoint that returns a blank or text-based response, and httpbin.org is a tried-and-true test domain. This also illustrates how the underlying client can be used without the defined object models. IMO tests should also illustrate usage.